README.md

```diff
@@ -1,8 +1,8 @@
-# Create-react-app with AWS Amplify Auth 
+# Create-react-app with AWS Amplify Auth
 
 This auth starter implements withAuthenticator HOC to provide a basic authentication flow for signing up signing in users as well as protected client side routing using AWS Amplify. Auth features: User sign up, User sign in, Multi-factor Authentication, User sign-out.
 
-[View Demo](https://master.d2ka7y7551sk8n.amplifyapp.com/)
+[View Docs And Demo](https://ui.docs.amplify.aws/components/authenticator)
```

Comment:
I wonder if instead of having the demo here, we instead have a link to the UI docs that has a demo on it too.